src/main/java/org/opensearch/security/securityconf/impl/v7/InternalUserV7.java

```diff
@@ -53,6 +53,8 @@
         private Map<String, String> attributes = Collections.emptyMap();
         private String description;
         private List<String> opendistro_security_roles = Collections.emptyList();
+        // default tenant set to global-tenant
+        private String saved_tenant = "global-tenant";
```

Comment:
Should `saved_tenant` be `final`?

Reply:
Yes; changed

Reply:
savedTenant?

Reply:
should be `saved_tenant` to be consistent with existing code (e.g. `opendistro_security_roles`)